src/components/Forms/UserNamForm.tsx

```diff
@@ -35,6 +35,8 @@ const UserNameForm = () => {
     userName && navigate(`/${userName}`);
   };
 
+  const isUserNameEmpty = userName.trim().length === 0;
```

Comment:
Instead of checking `length === 0 ` you can directly use `!userName` as a value inside tooltip disabled attribute.

Reply:
Done